spec/features/venues/view_venue_details_spec.rb

```diff
@@ -21,4 +21,20 @@
     expect(page).to have_content(venue.address)
     expect(page).to have_content(venue.capacity)
   end
+
+  scenario 'If T accessible, user should see message: T is nearby' do
+    venue = FactoryGirl.create(:venue, t_accessible: true)
+
+    visit venue_path(venue)
+
+    expect(page).to have_content(venue.display_t_accessible)
```

Comment:
You could save the string in a let statement at the beginning then you'll only have to change that one place and it will cascade or all other tests within. `let(:t_is_accessible_string') { "T is accessible" }` then expect page to have content t is accessible string. That's one option. If you want to roll with it the way it is for the purpose of flexibility later on, given that you have the method unit tested, that will be okay too. Respond with which way you want to go